rep-0127.rst

```diff
@@ -5,7 +5,7 @@ Status: Accepted
 Type: Standards Track
 Content-Type: text/x-rst
 Created: 12-Sep-2012
-Post-History: 22-Sep-2012
+Post-History: 31-Mar-2017
```

Comment:
Shouldn't this be appended rather than replaced? For example: https://raw.githubusercontent.com/ros-infrastructure/rep/master/rep-0003.rst